src/main/java/lotto/domain/InsightResult.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class InsightResult {
```

Comment:
일급 컬렉션 👍 복수 명사를 클래스 이름으로 사용하면 컬렉션임을 더 쉽게 표현할 수 있겠죠?

```suggestion
public class InsightResults {
```